Dockerfile

```diff
@@ -88,7 +88,7 @@ RUN echo gem '"rdoc"' > bundler.d/container.rb && echo gem '"tzinfo-data"' >> bu
 RUN date -u > BUILD_TIME
 
 # Start the main process.
-CMD bundle exec bin/rails server
+CMD bundle exec bin/rails server -b 0.0.0.0
```

Comment:
why is this required? I didn't see it was needed before?

Reply:
we override it in compose:
https://github.com/theforeman/foreman/blob/develop/docker-compose.yml#L24

and without that, you can't publish :3000 on the host, as docker proxy will do the request from 172.16.x.y and that won't be able to reach rails :(

Reply:
so yeah, it's probably not needed after https://github.com/theforeman/foreman/pull/7540 as @ekohl points out, but we should be consistent with the compose file (and maybe not override the command there?)

Reply:
After #7540, I think this is redundant.

Reply:
probably.

@ohadlevy shall we drop the `command` from the compose file? I'd prefer not to have multiple definitions of the same thing, makes tracing errors harder